roles/mesos/tasks/follower.yml

```diff
@@ -78,6 +78,15 @@
   tags:
     - mesos
 
+- name: set follower resources
+  sudo: yes
+  copy:
+    dest: /etc/mesos-slave/resources
+    content: "{{ mesos_resources }}"
+  notify: restart mesos follower
```

Comment:
@stevendborrelli when I was *changing* ports earlier it needed to be done, the follower wouldn't start before I removed the metadata. So if the end user changes the available ports, it will need to be done. Might as well remove it on every change to be safe.